_posts/2022-04-11-1.15-announcement.md

```diff
@@ -0,0 +1,375 @@
+at this point, because it was implemented following 
+very closely the new connector interfaces offering asynchronous functionality as well 
+as end-to-end semantics. This sink will act as a role model going forward.
+
+
+### Scala-free flink
+
+There was already a lengthy 
+[blogpost](https://flink.apache.org/2022/02/22/scala-free.html) by Seth Wiesman 
+in February explaining the ins and outs of why Scala users can now use the Flink 
+Java API with any Scala version - including Scala 3. At the end it was continuing 
+on what was started in 1.14 - removing the Mesos integration -, Akka has been isolated, 
+the DataStream / DataSet Java API has been cleaned up and hid the Table API behind an 
+abstraction. There’s already a lot of traction in the community.
+
+
+### Other changes that will further enable the community
+
+Further work has been done on the 
+[connector testing framework](https://github.com/PatrickRen/flink/tree/master/flink-test-utils-parent/flink-connector-testing), 
+if you want to contribute a connector or improve one you should definitely have a 
+look at it. One of the probably long awaited features has been added, the 
+[CSV format](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/connectors/datastream/formats/csv/) 
+and the [small file compaction](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/connectors/datastream/filesystem/#compaction) 
+in the unified sink interface has been implemented. The sink API has been upgraded 
+to [version 2](https://github.com/apache/flink/blob/master/flink-core/src/main/java/org/apache/flink/api/connector/sink2/StatefulSink.java)
+and we’d encourage every connector maintainer to upgrade this version.
+
+
+## To sum it up
+
+Apache Flink is now easier to operate, took another step towards aligning stream and 
+batch processing, became more accessible through some improvements in the SQL components 
+and integrates better with other technologies.Besides that it is worth mentioning that 
+the community - while working on everything listed above - has set up a new home for 
+the [CDC connectors](https://ververica.github.io/flink-cdc-connectors/release-2.1/index.html), 
+the [connector repository](https://cwiki.apache.org/confluence/display/FLINK/Connectors) 
+will be externalized 
+([once again see the elasticsearch sink as a first mover](https://github.com/apache/flink-connector-elasticsearch/)) 
+and there is now a 
+[Kubernetes operator](https://nightlies.apache.org/flink/flink-kubernetes-operator-docs-main/) 
+([announcement blogpost](https://flink.apache.org/news/2022/04/03/release-kubernetes-operator-0.1.0.html) 
+maintained by the community as well.
+
+Going forward the community will continue focussing on making Apache Flink a true 
+unified stream and batch processor as well as better integrating into the cloud native 
+ecosystem that we see ever evolving and thriving.
+
+
+## Upgrade Notes
```

Comment:
```suggestion
## Notes on Upgrading
```

Reply:
i think "Upgrade Notes" is quite commonly used.